gen-ai/orchestrator-server/src/main/python/tock-llm-indexing-tools/index_documents.py

```diff
@@ -164,23 +173,54 @@ async def index_documents(args):
         index_name=index_name,
         embedding_function=embeddings
     )
-    await vector_store_factory.check_vector_store_connection()
+    # await vector_store_factory.check_vector_store_connection()
     vector_store = vector_store_factory.get_vector_store()
 
     await embedding_and_indexing(splitted_docs, vector_store)
 
-    # Return indexing details
-    return index_name, session_uuid, len(docs), len(splitted_docs)
+    return IndexingDetails(
```

Comment:
Good to have a clean format it could be used to output this script as json format in an S3 bucket for instance when the ingestion is finished so that the pipeline could easily fetch the indexing session ID